src/pages/o/[orgId]/events/[eventId].tsx

```diff
@@ -0,0 +1,64 @@
+import { 
+    GetServerSideProps,
+    NextPageContext,
+} from 'next';
+
+export const getServerSideProps : GetServerSideProps = async (context : NextPageContext) => {
+    const { orgId, eventId } = context.params;
+    let props;
+
+    try {
+        const cRes = await fetch(`http://api.zetk.in/v1/orgs/${orgId}/campaigns`);
+        const cData = await cRes.json();
+        const oRes = await fetch(`https://api.zetk.in/v1/orgs/${orgId}`);
+        const oData = await oRes.json();
+
+        let allEventsData = [];
+        let eventData = [];
+
+        for (const obj of cData.data) {
+            const eventsRes = await fetch(`https://api.zetk.in/v1/orgs/${orgId}/campaigns/${obj.id}/actions`);
+            const campaignEvents = await eventsRes.json();
+            allEventsData = allEventsData.concat(campaignEvents.data);
+            eventData = allEventsData.find(event => event.id == eventId);
+            if (eventData) {
+                break;
+            }
+        }
+
```

Comment:
This point may be reached without `eventData` having been found, simply because it doesn't exist. When that happens, `eventData` will be undefined, and that should result in a `notFound` scenario like the one explained in the NEXT docs.